sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, endpoint, credential, **kwargs):
         authentication_policy = get_authentication_policy(credential)
         self._client = FormRecognizer(
             endpoint=endpoint,
-            credential=credential,
+            credential=credential,  # type: ignore
```

Comment:
What is the typing issue here?

Reply:
error: Argument "credential" to "FormRecognizerClient" has incompatible type "Union[AzureKeyCredential, TokenCredential]"; expected "TokenCredential"